fdbclient/BlobGranuleFiles.cpp

```diff
@@ -311,8 +311,9 @@ struct IndexBlockRef {
 		    getEncryptAuthTokenMode(EncryptAuthTokenMode::ENCRYPT_HEADER_AUTH_TOKEN_MODE_SINGLE),
 		    BlobCipherMetrics::BLOB_GRANULE);
 		BlobCipherEncryptHeaderRef headerRef;
-		buffer =
-		    encryptor.encrypt(serializedBuff.contents().begin(), serializedBuff.contents().size(), &headerRef, arena);
+		buffer = serializedBuff;
```

Comment:
nit: do we need `buffer` variable?

Reply:
Yeah i don't think so but it'll be annoying to change it here. I'll remember this in a followup pr